### DISCUSSION TITLE
Need `bulletInActiveClass` for Tailwind CSS

### DISCUSSION THREAD

**Discussion by dninomiya** [nolimits4web/swiper#5869] [Ideas]
Example

```jsx
pagination={{
  clickable: true,
  bulletClass:
    'w-2.5 h-2.5 block rounded-full cursor-pointer',
  bulletActiveClass: 'bg-blue-500',
  bulletInActiveClass: 'bg-slate-500', // idea
  el: '#pagination',
}}
```

## Why?

When controlling bullet color with tailwind css, depending on the color combination, the color set in bulletClass has priority over bulletActiveClass and bulletActiveClass is not reflected.

It is possible to deal with it even now by using important (`!bg-blue-500`), but I would like to avoid it if possible.

**Comment by jaads** [comment]
Do you have any idea regarding #5969 ?
